helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -58,9 +61,15 @@ controller:
   role:
     name: nginx-ingress-role
 
-  # Sets the NodePorts that maps to the Ingress' ports 80 (http) and 443 (https).
   service:
     enabled: true
+    # type applies only to AWS provider:
+    # - external (default)
+    # - internal
+    # ...
+    type: external
+    # Sets the NodePorts that maps to the Ingress' ports 80 (http) and 443 (https).
+    # nodePort service type ig not created for AWS provider as LoadBalancer is used there.
```

Comment:
```suggestion
    # nodePort service type is not created for AWS provider as LoadBalancer is used there.
```

Reply:
OK cool. See https://github.com/giantswarm/giantswarm/issues/4795#issuecomment-542209785 we should make sure the node port service is not created by Ignition. We can do this via the k8scloudconfig param.